scss/_variables.scss

```diff
@@ -299,17 +299,29 @@ $table-dark-color:           $body-bg !default;
 //
 // For each of Bootstrap's buttons, define text, background and border color.
 
-$input-btn-padding-y:       .375rem !default;
-$input-btn-padding-x:       .75rem !default;
-$input-btn-line-height:     $line-height-base !default;
+$input-padding-y:       .375rem !default;
```

Comment:
Can you please move `$input-*` variables under "Forms" (after line 331).

Reply:
@andresgalante good correct.